### ISSUE TITLE
searchitem command already used by WorldEdit / FAWE

### ISSUE THREAD

**Issue by taupemoi** [JEFF-Media-GbR/Spigot-InvUnloadPlus#32]
WorldEdit / FAWE already has a searchitem command which takes precedence. But /search works fine.

(Tested with FAWE 1.16-302 & InvUnload 4.8.2 on Paper 1.16.1 build 101.)

**Comment by mfnalex** [comment]
Yes I know. Do you have alternative command name suggestions? After all, it still possible to use every command via /[plugin]:[command]

**Comment by taupemoi** [comment]
Maybe `chestsearch`?

**Comment by mfnalex** [comment]
I will add this alias :)
